tests/stdlib/tosproc.nim

```diff
@@ -4,60 +4,60 @@ joinable: false
 
 #[
 joinable: false
-because it'd need cleanup up stdout
+because it has to execute itself
```

Comment:
I think this would be better if the runner supported an array argument sets to pass a test so it could call it repeatedly instead. Although in this case we'd need more for the whole input output bit? Also scope creep.

Reply:
This test tests `osproc`, and it executes itself with `osproc.*` to run its subprogram to verify behaviors. It has nothing to do with having to run with different arguments :P

Reply:
Wait, the sub-programs are itself, right? Or did I read that wrong?

Anyhow, I was considering it for each of the permutation that it would run, would still need to have other processes and all that. Just that the reporting could be a little bit more fine grained.

Reply:
Ah yea, we can certainly add a runner <-> test IPC in the future.